docs/source/ecf.rst

```diff
@@ -1447,8 +1448,9 @@ timescales. It is also possible to plot
 'fn' (the nightside flux from a sinusoidal phase curve),
 'pc_offset' (the sinusoidal offset of the phase curve),
 'pc_amp' (the sinusoidal amplitude of the phase curve),
-'offset_order1' or 'offset_order2' (the first or second order sinusoidal offset of the phase curve), and
-'amp_order1' or 'amp_order2' (the first or second order sinusoidal amplitude of the phase curve).
+'offset_order1' or 'offset_order2' (the first or second order sinusoidal offset of the phase curve),
+'amp_order1' or 'amp_order2' (the first or second order sinusoidal amplitude of the phase curve), and
+'morning' or 'evening' (the morning and evening terminators when using Harmonica).
```

Comment:
We likely want to use more descriptive names and documentation here, since `'morning'` and `'evening'` could equally appear to be referring to the morning/evening radii or the morning/evening hemispherical emission spectra

Reply:
Done, but let me know if you have something else in mind.

Reply:
Looks good to me